pkg/controller/metrics/metrics.go

```diff
@@ -133,6 +133,13 @@ func getAccountFromLabels(labels map[string]string) string {
 	return account
 }
 
+func getAuditFromSpec(spec models.KlusterSpec) string {
+	if spec.Audit == nil {
+		return "NA"
```

Comment:
I suggest to set it to the empty string `""` instead

Reply:
I decided to go for "NA" for consistency:
https://github.com/sapcc/kubernikus/blob/master/pkg/controller/metrics/metrics.go#L120-L134

There is situations where the alertmanager evaluates an empty label matching a missing label:
https://prometheus.io/docs/alerting/latest/configuration/#inhibit_rule

Reply:
I am happy to commit either one of them, you decide @databus23  :-)